### ISSUE TITLE
The attention score in model_sparse

### ISSUE THREAD

**Issue by ty4b112** [seongjunyun/Graph_Transformer_Networks#10]
To find meta-paths with high attention scores learnt by GTNs, I print the attention scores in main.py (denoted as Ws in line 100) and main_sparse.py (denoted as _ in line 127).
I run your code with: "python main_sparse.py --dataset IMDB --num_layers 3 --adaptive_lr true".
Surprisingly, it seems that the model did not train the weight of each GTConv at all, the weights after softmax are always [0.2, 0.2, 0.2, 0.2, 0.2].

**Comment by seongjunyun** [comment]
Hi, 
We ran our code by the same command you did and upload the log here.
![image](https://user-images.githubusercontent.com/19612545/75418387-aa83d180-5976-11ea-8016-3bac126e0894.png)

The weight of each GTConv kept changing during the training process in this log.
If you modified something in our code, then please tell me about it.

**Comment by ty4b112** [comment]
Hi,
I clone the latest version of your code and do not change any of your code except add "print(_)“ after line 119 in main_sparse.py, but the results are still the same.
My dependencies are as follows:
CUDA Version        10.1.168
python              3.7.4
torch               1.4.0    
torch-geometric     1.4.2    
torch-scatter       2.0.4    
torch-sparse        0.5.1 
![图片](https://user-images.githubusercontent.com/12560113/75432000-b2e50800-5988-11ea-9432-1cd5618d3cc0.png)

**Comment by changym3** [comment]
> Hi,
> I clone the latest version of your code and do not change any of your code except add "print(_)“ after line 119 in main_sparse.py, but the results are still the same.
> My dependencies are as follows:
> CUDA Version 10.1.168
> python 3.7.4
> torch 1.4.0
> torch-geometric 1.4.2
> torch-scatter 2.0.4
> torch-sparse 0.5.1
> ![图片](https://user-images.githubusercontent.com/12560113/75432000-b2e50800-5988-11ea-9432-1cd5618d3cc0.png)

Hey! I meet this problem too. And I find that the tensors returned from `spspmm` will have the value `None` in `grad_fn`. In fact it's the due to the version of the `torch_sparse`. The package has removed the autograd support for the function `spspmm`.  [Detail here](https://github.com/rusty1s/pytorch_sparse/issues/45). You could just reinstall `torch_sparse` below `0.4.4` release. You could get the correct attention score.
